src/Gemfile

```diff
@@ -0,0 +1,3 @@
+source 'https://rubygems.org'
```

Comment:
file no more needed?

Reply:
No, this file is used to install Color and Git modules and since they aren't required for the moment i've removed the Gemfiles.